File: Programming Fundamentals/C#/01.Basic Syntax, Conditional Statements and Loops/[MoreExercises]/P01_SortNumbers/P01_SortNumbers.cs
namespace P01_SortNumbers
{
    using System;

    class P01_SortNumbers
    {
        static void Main(string[] args)
        {
            int num1 = int.Parse(Console.ReadLine());
            int num2 = int.Parse(Console.ReadLine());
            int num3 = int.Parse(Console.ReadLine());

            int max = Math.Max(Math.Max(num1, num2), num3);
            int min = Math.Min(Math.Min(num1, num2), num3);
            int mid = (num1 + num2 + num3) - (max + min);

            Console.WriteLine(max);
            Console.WriteLine(mid);
            Console.WriteLine(min);
        }
    }
}

File: Programming Fundamentals/JS/01.Basic Syntax, Conditional Statements and Loops/[MoreExercises]/p01_SortNumbers.js
function solve(num1, num2, num3) {
  let max = Math.max(Math.max(num1, num2), num3);
  let min = Math.min(Math.min(num1, num2), num3);
  let mid = (num1 + num2 + num3) - (max + min);

  console.log(max);
  console.log(mid);
  console.log(min);
}

// Don't copy the calling of the function in judge, you won't get any points, just the code above
solve(2, 1, 3);
solve(-2, 1, 3);
solve(0, 0, 2);
